core/src/edu/asu/c3simulator/screens/PreMarketProducts.java

```diff
@@ -0,0 +1,214 @@
+		Table testTable = new Table();
+		ProductDesigns newProductPopupWindow = new ProductDesigns(testTable,
+				stage, skin);
+		// button for creating new products
+		Table createNewTable = new Table();
+
+		FileHandle img = Gdx.files.internal("images/Create-new.png");
+		Image newProductButton = new Image(new Texture(img));
+		newProductButton.addListener(new ClickListener() {
+			@Override
+			public void clicked(InputEvent event, float x, float y)
+			{
+				newProductPopupWindow.displayProductDesignOptions();
+			}
+		});
+		productTable.add(testTable);
+		createNewTable.add(newProductButton);
+		createNewTable.row();
+		createNewTable.add(new Label("Create new", skin));
+		productTable.add(createNewTable);
+
+		ScrollPane chosenTasksScroll = new ScrollPane(productTable, skin);
+
+		Table leftSection = new Table();
+		leftSection.debug();
+		leftSection.add(text).fillX().align(Align.center);
+		leftSection.row();
+		leftSection.add(chosenTasksScroll).expand().fill().center();
+		return leftSection;
+	}
+
+	/**
+	 * Creates a table with detailed information about the pre market products
+	 * in a panel that the can be scrolled through.
+	 * 
+	 * @return Table
+	 */
+	private Actor ProductInfo()
+	{
+
+		// TODO: backend. remove product interface, implement actual product
+		// class,
+		// replace selected employees in dropdown with real employees,
+		// refresh product info
+		ProductInfoPanel productInfoPanel = new ProductInfoPanel(skin);
+		Table productInfo = productInfoPanel.getProductInfo();
+		// TODO: add functionality so buttons will open appropriate windows when
+		// clicked.
+		// TODO: add functionality so buttons will open appropriate windows when
+		// clicked.
+		Table productButtons = new Table();
+		FileHandle edit = Gdx.files.internal("images/edit.png");
+		productButtons.add(new Image(new Texture(edit))).left().size(100);
+		FileHandle launch = Gdx.files.internal("images/launch.png");
+		productButtons.add(new Image(new Texture(launch))).right().size(100)
+				.space(Value.percentWidth(1.4f));
+		productInfo.add(productButtons).left();
+		ScrollPane productPane = new ScrollPane(productInfo.top(), skin);
+		return productPane;
+	}
+
+	@Override
+	public void dispose()
+	{
+		stage.dispose();
+		this.game = null;
+	}
+
+	@Override
+	public void hide()
+	{
+		Gdx.input.setInputProcessor(null);
+	}
+
+	@Override
+	public void pause()
+	{
+		throw new UnsupportedOperationException(
+				"The method is not implemented yet.");
+	}
+
+	@Override
+	public void render(float delta)
+	{
+		Gdx.gl.glClear(GL20.GL_COLOR_BUFFER_BIT);
+		stage.act(Gdx.graphics.getDeltaTime());
+		stage.draw();
+	}
+
+	@Override
+	public void resize(int width, int height)
+	{
+		stage.getViewport().update(width, height);
+	}
+
+	@Override
+	public void resume()
+	{
+		throw new UnsupportedOperationException(
+				"The method is not implemented yet.");
+	}
+
+	@Override
+	public void show()
+	{
+		Gdx.input.setInputProcessor(stage);
+	}
+}
```

Comment:
This class has a lot of mistakes similar or identical to other cases. I am postponing the remainder of the review of this class until it is refactored to address the relative issues mentioned elsewhere in this review.